pom.xml

```diff
@@ -81,6 +91,33 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.teavm</groupId>
+                <artifactId>teavm-maven-plugin</artifactId>
+                <version>0.9.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.teavm</groupId>
+                        <artifactId>teavm-classlib</artifactId>
+                        <version>0.9.2</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                        <phase>process-classes</phase>
+                        <configuration>
+                            <mainClass>com.shapesecurity.salvation2.Salvation2</mainClass>
+                            <minifying>true</minifying>
```

Comment:
No need to minify; better the code be at least somewhat readable.

Reply:
It doesn't generate very readable code, regardless of minification. One would definitely need to generate source maps to debug effectively.

Reply:
Yeah ok.